### PR TITLE
Adam profile (Normal Mode)

### PR DESCRIPTION
**Pull request by adamwage**
Finished normal mode for the assignment 👍 🤓 😬!!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>3.1 - Github Profile</title>
+    <link rel="stylesheet" href="styles/main.css">
+    <script src="javascripts/main.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <!-- name, blog, location, email, avatar_url, html_url, list all of your repositories -->
```
Comment (jasonvanderslice) [inline index.html]: 👍

**Comment by jasonvanderslice** [comment]
Your consistency is 🔑 - keep up the quality of code (good to merge)

**Comment by adamwage** [comment]
Thanks!! I approve the gif ^^

**Pull request merged**